Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -478,6 +478,79 @@ public struct MongoDatabase {
         return self._client.operationExecutor.execute(operation, client: self._client, session: session)
     }
 
+    /**
+     * Runs an aggregation framework pipeline against this database for pipeline stages that do not require an
+     * underlying collection, such as `$currentOp` and `$listLocalSessions`.
+     *
+     * - Parameters:
+     *   - pipeline: an `[Document]` containing the pipeline of aggregation operations to perform.
```

Comment:
```suggestion
     *   - pipeline: an `[BSONDocument]` containing the pipeline of aggregation operations to perform.
```

same comment applies to other docstrings you've added.

I think this is a copy paste from the docs from the `MongoCollection` `aggregate` methods which I guess we forgot to update back when we renamed this type. would you mind fixing those ones too?